game/lib/journeys/journeys_page.dart

```diff
@@ -160,54 +170,66 @@ class _JourneysPageState extends State<JourneysPage> {
 
                       final challengeLocation = challenge.location?.name ?? "";
                       final challengeName = challenge.name ?? "";
+                      final eventDifficulty = event.difficulty?.name ?? "";
+                      final String eventName = event.name ?? "";
```

Comment:
Could these be named Journey Difficulty and Journey Name for easier clarification of what they are referring to (since an event can almost be a journey or challenge depending on the number of challenges)